src/v/config/configuration.cc

```diff
@@ -3792,6 +3792,23 @@ configuration::configuration()
       {.visibility = visibility::user},
       std::nullopt,
       &validate_non_empty_string_opt)
+  , iceberg_backlog_controller_p_coeff(
+      *this,
+      "iceberg_backlog_controller_p_coeff",
+      "Proportional coefficient for the Iceberg backlog controller. Number of "
+      "shares assigned to the datalake scheduling group will be proportional "
+      "to the backlog size error.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      -0.0003)
+  , iceberg_target_backlog_size(
+      *this,
+      "iceberg_target_backlog_size",
+      "Average size of the datalake translation backlog, per partition, that "
+      "the "
+      "backlog controller will try to maintain.",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      5_MiB,
```

Comment:
I was initially considering whether the default size should be slightly larger than 5 MiB. However, a smaller default has the advantage of smaller proportional changes to scheduling weights. This allows the system to correct itself more quickly than with a larger default (e.g., 1 GiB), where weight changes would be drastic and throttling more significant (bigger sawtooth).  Therefore, we likely prefer smaller values for faster system adjustment, am I thinking about it correctly?

Reply:
hmm ya, batch cache hits reasoning make sense.